docs/SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,102 @@
+Supported Platforms
+===================
+Telegraf is a cross-platform application. This doc helps define which operating systems, distributions, and releases Telegraf supports.
+
+Telegraf is supported on Linux, FreeBSD, Windows, and macOS. It is written in Go which supports these operating systems and more. Telegraf may work on Go's other operating systems and users are welcome to build their own binaries for them. Bug reports should be submitted only for supported platforms.
+
+https://golang.org/doc/install/source
+
+https://github.com/golang/go/wiki/MinimumRequirements#operating-systems
+
+Linux
+-----
+Telegraf intent: *Support latest versions of the most popular distributions*
+
+https://distrowatch.com/dwres.php?resource=major
+
+Telegraf supports RHEL, Fedora, Debian, and Ubuntu. InfluxData provides package repositories for these distributions. Instructions for using the package repositories can be found on [docs.influxdata.com](https://docs.influxdata.com/telegraf/v1.16/introduction/installation/) Bug reports shuld be submitted only for supported distributions and releases.
```

Comment:
use `latest` in docs link will link to the latest version: https://docs.influxdata.com/telegraf/latest/introduction/installation/

Reply:
Add a `.` after the link before `Bug reports`

Reply:
spelling: shuld >> should